### ISSUE TITLE
Controller reports invalid but no details

### ISSUE THREAD

**Issue by paulcarlton-ww** [weaveworks/tf-controller#517]
I have a terraform that runs ok from command line but tf-controller reports invalid

```
kubectl logs -f -n flux-system tf-controller-7cdcc78b6f-sxscs | grep "^{" | jq -r '. | select ( .name == "beta-cnfs-core")'
{
  "level": "info",
  "ts": "2023-02-08T12:59:46.082Z",
  "msg": ">> Started Generation: 1",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z"
}
{
  "level": "info",
  "ts": "2023-02-08T12:59:46.095Z",
  "msg": "getting source",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z"
}
{
  "level": "info",
  "ts": "2023-02-08T12:59:46.095Z",
  "msg": "before lookup runner: checking ready condition",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z",
  "ready": "&Condition{Type:Ready,Status:Unknown,ObservedGeneration:0,LastTransitionTime:2023-02-08 12:58:53 +0000 UTC,Reason:Progressing,Message:Reconciliation in progress,}"
}
{
  "level": "info",
  "ts": "2023-02-08T12:59:46.095Z",
  "msg": "trigger namespace tls secret generation",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z"
}
{
  "level": "info",
  "ts": "2023-02-08T12:59:46.548Z",
  "msg": "show runner pod state: ",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z",
  "state": "must-be-deleted"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:16.641Z",
  "msg": "runner is running",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:16.641Z",
  "msg": "setting up terraform",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:18.649Z",
  "msg": "write backend config: ok",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:18.650Z",
  "msg": "new terraform",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z",
  "workingDir": "/tmp/flux-system-beta-cnfs-core/cluster-templates/leaf-cluster-core"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:18.667Z",
  "msg": "generate vars from tf: ok",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:18.667Z",
  "msg": "generated var files from spec",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:18.670Z",
  "msg": "generate template: ok",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:18.670Z",
  "msg": "generated template",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z"
}
{
  "level": "error",
  "ts": "2023-02-08T13:00:18.980Z",
  "msg": "error in terraform setup",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z",
  "error": "error running Init: rpc error: code = Internal desc = configuration is invalid"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:18.989Z",
  "msg": "clean up dir: ok",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z"
}
{
  "level": "error",
  "ts": "2023-02-08T13:00:19.001Z",
  "msg": "Reconciliation failed after 33.427852489s, next try in 20s",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "fa3d9690-ad07-4bf4-9af9-98656e34c32c",
  "reconciliation-loop-id": "1708d596-6f93-4651-a262-7d03da33058f",
  "start-time": "2023-02-08T12:59:45.573Z",
  "revision": "main/c8a95efa0adf802c49b4f4360ddbd9fea657a411",
  "error": "error running Init: rpc error: code = Internal desc = configuration is invalid"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:44.019Z",
  "msg": ">> Started Generation: 1",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "29121ca9-55b5-4179-97a0-7220a1eda05a",
  "reconciliation-loop-id": "65245530-e3cb-4467-9609-55d417a20c7c",
  "start-time": "2023-02-08T13:00:44.019Z"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:44.019Z",
  "msg": "getting source",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "29121ca9-55b5-4179-97a0-7220a1eda05a",
  "reconciliation-loop-id": "65245530-e3cb-4467-9609-55d417a20c7c",
  "start-time": "2023-02-08T13:00:44.019Z"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:44.019Z",
  "msg": "before lookup runner: checking ready condition",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "29121ca9-55b5-4179-97a0-7220a1eda05a",
  "reconciliation-loop-id": "65245530-e3cb-4467-9609-55d417a20c7c",
  "start-time": "2023-02-08T13:00:44.019Z",
  "ready": "&Condition{Type:Ready,Status:False,ObservedGeneration:0,LastTransitionTime:2023-02-08 13:00:18 +0000 UTC,Reason:TFExecInitFailed,Message:error running Init: rpc error: code = Internal desc = configuration is invalid,}"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:44.019Z",
  "msg": "before lookup runner: updating status",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "29121ca9-55b5-4179-97a0-7220a1eda05a",
  "reconciliation-loop-id": "65245530-e3cb-4467-9609-55d417a20c7c",
  "start-time": "2023-02-08T13:00:44.019Z",
  "ready": "&Condition{Type:Ready,Status:False,ObservedGeneration:0,LastTransitionTime:2023-02-08 13:00:18 +0000 UTC,Reason:TFExecInitFailed,Message:error running Init: rpc error: code = Internal desc = configuration is invalid,}"
}
{
  "level": "info",
  "ts": "2023-02-08T13:00:44.032Z",
  "msg": "before lookup runner: updated status",
  "controller": "terraform",
  "controllerGroup": "infra.contrib.fluxcd.io",
  "controllerKind": "Terraform",
  "Terraform": {
    "name": "beta-cnfs-core",
    "namespace": "flux-system"
  },
  "namespace": "flux-system",
  "name": "beta-cnfs-core",
  "reconcileID": "29121ca9-55b5-4179-97a0-7220a1eda05a",
  "reconciliation-loop-id": "65245530-e3cb-4467-9609-55d417a20c7c",
  "start-time": "2023-02-08T13:00:44.019Z",
  "ready": "&Condition{Type:Ready,Status:Unknown,ObservedGeneration:0,LastTransitionTime:2023-02-08 13:00:44.019336972 +0000 UTC m=+106.731534051,Reason:Progressing,Message:Reconciliation in progress,}"
}
```
Terraform object

```
kubectl get terraforms.infra.contrib.fluxcd.io -n flux-system beta-cnfs-core  -o yaml
apiVersion: infra.contrib.fluxcd.io/v1alpha1
kind: Terraform
metadata:
  creationTimestamp: "2023-02-08T11:44:10Z"
  finalizers:
  - finalizers.tf.contrib.fluxcd.io
  - tf.dependency.of.beta-cnfs-cnfs
  generation: 1
  labels:
    kustomize.toolkit.fluxcd.io/name: gitops-clusters
    kustomize.toolkit.fluxcd.io/namespace: flux-system
    templates.weave.works/template-name: eks-leaf-cluster-cnfs
    templates.weave.works/template-namespace: default
  name: beta-cnfs-core
  namespace: flux-system
  resourceVersion: "810077"
  uid: 439928d0-ed24-4b34-9a26-9a6b037c4ee0
spec:
  alwaysCleanupRunnerPod: true
  approvePlan: auto
  backendConfig:
    customConfiguration: |
      backend "s3" {
        bucket         = 'ww-20276-749339757188-eu-west-1-tf-state'
        key            = "leaf-clusters/default/beta-cnfs/leaf-cluster-core/terraform.tfstate"
        region         = 'eu-west-1'
        encrypt        = true
        dynamodb_table = 'ww-20276-749339757188-eu-west-1-tf-state'
      }
  destroyResourcesOnDeletion: true
  disableDriftDetection: false
  force: false
  interval: 1h
  parallelism: 0
  path: ./cluster-templates/leaf-cluster-core
  refreshBeforeApply: false
  retryInterval: 20s
  runnerTerminationGracePeriodSeconds: 30
  serviceAccountName: tf-runner
  sourceRef:
    kind: GitRepository
    name: terraform
    namespace: flux-system
  storeReadablePlan: none
  vars:
  - name: cluster_name
    value: beta-cnfs
  - name: cluster_version
    value: "1.23"
  - name: region
    value: eu-west-1
  - name: vpc_cidr
    value: 10.0.0.0/16
  - name: public_subnet_count
    value: "3"
  - name: private_subnet_count
    value: "3"
  workspace: default
  writeOutputsToSecret:
    name: beta-cnfs-core-outputs
status:
  conditions:
  - lastTransitionTime: "2023-02-08T13:03:31Z"
    message: Reconciliation in progress
    reason: Progressing
    status: Unknown
    type: Ready
  lastAttemptedRevision: main/c8a95efa0adf802c49b4f4360ddbd9fea657a411
  observedGeneration: 1
```

Running latest version of tf-controller, or at least I thought  I was but it does not seem to be picking up my value

```
kubectl get pod -n flux-system tf-controller-7cdcc78b6f-sxscs -o yaml
apiVersion: v1
kind: Pod
metadata:
  annotations:
    container.seccomp.security.alpha.kubernetes.io/tf-controller: runtime/default
    kubernetes.io/psp: eks.privileged
  creationTimestamp: "2023-02-08T12:58:55Z"
  generateName: tf-controller-7cdcc78b6f-
  labels:
    app.kubernetes.io/instance: tf-controller
    app.kubernetes.io/name: tf-controller
    pod-template-hash: 7cdcc78b6f
  name: tf-controller-7cdcc78b6f-sxscs
  namespace: flux-system
  ownerReferences:
  - apiVersion: apps/v1
    blockOwnerDeletion: true
    controller: true
    kind: ReplicaSet
    name: tf-controller-7cdcc78b6f
    uid: f680b84e-902a-404f-95c2-05ab2dffabfd
  resourceVersion: "807434"
  uid: 4914b2b9-6430-42ed-9f53-15741726fb19
spec:
  containers:
  - args:
    - --watch-all-namespaces
    - --log-level=info
    - --log-encoding=json
    - --enable-leader-election
    - --concurrent=24
    - --ca-cert-validity-duration=168h0m
    - --cert-rotation-check-frequency=30m0s
    - --cert-validity-duration=6h0m
    - --runner-creation-timeout=5m0s
    - --runner-grpc-max-message-size=30
    - --events-addr=http://notification-controller.flux-system.svc.cluster.local./
    - --kube-api-qps=50
    - --kube-api-burst=100
    command:
    - /sbin/tini
    - --
    - tf-controller
    env:
    - name: RUNTIME_NAMESPACE
      valueFrom:
        fieldRef:
          apiVersion: v1
          fieldPath: metadata.namespace
    - name: RUNNER_POD_IMAGE
      value: ghcr.io/weaveworks/tf-runner:v0.14.0-rc.5
    image: ghcr.io/weaveworks/tf-controller:v0.13.1
    imagePullPolicy: IfNotPresent
    livenessProbe:
      failureThreshold: 3
      httpGet:
        path: /healthz
        port: healthz
        scheme: HTTP
      periodSeconds: 10
      successThreshold: 1
      timeoutSeconds: 1
    name: tf-controller
    ports:
    - containerPort: 8080
      name: http-prom
      protocol: TCP
    - containerPort: 9440
      name: healthz
      protocol: TCP
    readinessProbe:
      failureThreshold: 3
      httpGet:
        path: /readyz
        port: healthz
        scheme: HTTP
      periodSeconds: 10
      successThreshold: 1
      timeoutSeconds: 1
    resources:
      limits:
        cpu: "1"
        memory: 1Gi
      requests:
        cpu: 200m
        memory: 64Mi
    securityContext:
      allowPrivilegeEscalation: false
      capabilities:
        drop:
        - ALL
      readOnlyRootFilesystem: true
      runAsNonRoot: true
      runAsUser: 65532
      seccompProfile:
        type: RuntimeDefault
    terminationMessagePath: /dev/termination-log
    terminationMessagePolicy: File
    volumeMounts:
    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
      name: kube-api-access-d8fth
      readOnly: true
  dnsPolicy: ClusterFirst
  enableServiceLinks: true
  nodeName: ip-10-0-16-227.eu-west-1.compute.internal
  preemptionPolicy: PreemptLowerPriority
  priority: 0
  restartPolicy: Always
  schedulerName: default-scheduler
  securityContext:
    fsGroup: 1337
  serviceAccount: tf-controller
  serviceAccountName: tf-controller
  terminationGracePeriodSeconds: 10
  tolerations:
  - effect: NoExecute
    key: node.kubernetes.io/not-ready
    operator: Exists
    tolerationSeconds: 300
  - effect: NoExecute
    key: node.kubernetes.io/unreachable
    operator: Exists
    tolerationSeconds: 300
  volumes:
  - name: kube-api-access-d8fth
    projected:
      defaultMode: 420
      sources:
      - serviceAccountToken:
          expirationSeconds: 3607
          path: token
      - configMap:
          items:
          - key: ca.crt
            path: ca.crt
          name: kube-root-ca.crt
      - downwardAPI:
          items:
          - fieldRef:
              apiVersion: v1
              fieldPath: metadata.namespace
            path: namespace
status:
  conditions:
  - lastProbeTime: null
    lastTransitionTime: "2023-02-08T12:58:55Z"
    status: "True"
    type: Initialized
  - lastProbeTime: null
    lastTransitionTime: "2023-02-08T12:58:57Z"
    status: "True"
    type: Ready
  - lastProbeTime: null
    lastTransitionTime: "2023-02-08T12:58:57Z"
    status: "True"
    type: ContainersReady
  - lastProbeTime: null
    lastTransitionTime: "2023-02-08T12:58:55Z"
    status: "True"
    type: PodScheduled
  containerStatuses:
  - containerID: docker://cb656eaac3f5343c3728029a0ea48637eb9d8a6cda7527c6d0f230d5349edcf3
    image: ghcr.io/weaveworks/tf-controller:v0.13.1
    imageID: docker-pullable://ghcr.io/weaveworks/tf-controller@sha256:754df48b11e46d0bc88f452b90e9b59e5b050653d07df806ff39d15e5b4d1931
    lastState: {}
    name: tf-controller
    ready: true
    restartCount: 0
    started: true
    state:
      running:
        startedAt: "2023-02-08T12:58:57Z"
  hostIP: 10.0.16.227
  phase: Running
  podIP: 10.0.16.252
  podIPs:
  - ip: 10.0.16.252
  qosClass: Burstable
  startTime: "2023-02-08T12:58:55Z"
paulc:terraform paulc [sandbox]$ kubectl get helmreleases.helm.toolkit.fluxcd.io -n flux-system tf-controller -o yaml
apiVersion: helm.toolkit.fluxcd.io/v2beta1
kind: HelmRelease
metadata:
  creationTimestamp: "2023-02-07T10:48:16Z"
  finalizers:
  - finalizers.fluxcd.io
  generation: 3
  labels:
    kustomize.toolkit.fluxcd.io/name: infrastructure
    kustomize.toolkit.fluxcd.io/namespace: flux-system
  name: tf-controller
  namespace: flux-system
  resourceVersion: "805965"
  uid: 761f1036-07b4-409d-aca8-3640894721b3
spec:
  chart:
    spec:
      chart: tf-controller
      reconcileStrategy: ChartVersion
      sourceRef:
        kind: HelmRepository
        name: tf-controller
      version: 0.10.1
  install:
    crds: CreateReplace
    remediation:
      retries: 3
  interval: 10m0s
  upgrade:
    crds: CreateReplace
    remediation:
      retries: 3
  values:
    runner:
      grpc:
        maxMessageSize: 30
      image:
        tag: v0.14.0-rc.5
      serviceAccount:
        annotations:
          eks.amazonaws.com/role-arn: arn:aws:iam::749339757188:role/ww-sandbox2-wge-tf-controller
status:
  conditions:
  - lastTransitionTime: "2023-02-08T12:56:39Z"
    message: Release reconciliation succeeded
    reason: ReconciliationSucceeded
    status: "True"
    type: Ready
  - lastTransitionTime: "2023-02-08T12:56:39Z"
    message: Helm upgrade succeeded
    reason: UpgradeSucceeded
    status: "True"
    type: Released
  helmChart: flux-system/flux-system-tf-controller
  lastAppliedRevision: 0.10.1
  lastAttemptedRevision: 0.10.1
  lastAttemptedValuesChecksum: 8cf87406fbb2abf8ca9219b87d0cbd6ec5780c05
  lastReleaseRevision: 3
  observedGeneration: 3
```

**Comment by chanwit** [comment]
Thank you for reporting @paulcarlton-ww

**Comment by LappleApple** [comment]
Is this still an issue?

**Comment by LappleApple** [comment]
Is this still an issue?

**Comment by chanwit** [comment]
It would be fixed by https://github.com/weaveworks/tf-controller/pull/832

**Comment by LappleApple** [comment]
@paulcarlton-ww Could you please confirm the fix upon your return from vacation? Thanks so much.

**Comment by paulcarlton-ww** [comment]
Not seen this recently

**Comment by chanwit** [comment]
Hi @paulcarlton-ww 
The fix was in v0.16.0-rc.2.

**Comment by LappleApple** [comment]
@paulcarlton-ww Thanks for checking in. I'll close this, then.
